lighthouse-core/test/audits/metrics/largest-contentful-paint-test.js

```diff
@@ -7,39 +7,45 @@
 
 const LCPAudit = require('../../../audits/metrics/largest-contentful-paint.js');
 const defaultOptions = LCPAudit.defaultOptions;
+const constants = require('../../../config/constants.js');
 
 const trace = require('../../fixtures/traces/lcp-m78.json');
 const devtoolsLog = require('../../fixtures/traces/lcp-m78.devtools.log.json');
 
 const preLcpTrace = require('../../fixtures/traces/progressive-app-m60.json');
 const preLcpDevtoolsLog = require('../../fixtures/traces/progressive-app-m60.devtools.log.json');
 
-function generateArtifacts({trace, devtoolsLog, TestedAsMobileDevice, HostUserAgent}) {
+function generateArtifacts({trace, devtoolsLog, HostUserAgent}) {
   return {
     traces: {[LCPAudit.DEFAULT_PASS]: trace},
     devtoolsLogs: {[LCPAudit.DEFAULT_PASS]: devtoolsLog},
-    TestedAsMobileDevice,
     HostUserAgent,
   };
 }
 
-function generateContext({throttlingMethod}) {
-  const settings = {throttlingMethod};
-  return {options: defaultOptions, settings, computedCache: new Map()};
-}
+/**
+ * @param {LH.SharedFlagsSettings['formFactor']} formFactor
+ * @param {LH.SharedFlagsSettings['throttlingMethod']} throttlingMethod
+ */
+const getFakeContext = (formFactor, throttlingMethod) => ({
```

Comment:
are you still OK with keep the ones that used objects already to the object argument format? 😃

Reply:
yah.  i went ahead and changed the ones i added. no biggie. 1c3731e3f7a14d6c7d38ad55c56cb97dc8c40e15